.github/workflows/extbuild.yml

```diff
@@ -50,14 +52,14 @@ jobs:
           key: ${{ runner.os }}-${{ env.PIO_VERSION }}.pio
       - name: Build ParallelIO
         if: steps.cache-ParallelIO.outputs.cache-hit != 'true'
-        uses: NCAR/ParallelIO/.github/actions/parallelio_cmake@9390e30e29d4ebbfbef0fc72162cacd9e8f25e4e
+        uses: NCAR/ParallelIO/.github/actions/parallelio_cmake@pio2_6_0
```

Comment:
Is 2.6.0 the latest PIO version? I think we just requested 2.5.10 on wcoss2.

Reply:
Yes, it looks like 2.6.0 was released 2 weeks ago.

Reply:
The release note says

This release supports the netcdf integration layer using Dispatch version 5. We recommend netcdf-c v4.9.2 or newer to take advantage of this feature.

We have also added API's for netcdf4/hdf5 filters. These provide a full set of lossless and lossy compression techniques.

Reply:
I tested this PR using what is currently used for ufs, so I do not think it requires the more recent pio.